node/src/Node.hs

```diff
@@ -44,6 +45,7 @@ makeRandomAddress crypt = Address <$> C.randomBytes crypt 64
 
 main :: IO ()
 main = do
+  mapM_ (`hSetEncoding` utf8) [stdout, stdin, stderr]
```

Comment:
stdout and stdin are in binary mode (it's how the node communicates with the container) already, so I don't think setting these makes a difference. I guess it doesn't hurt.